microsoft-365/security/mtp/advanced-hunting-overview.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Overview - Advanced hunting
-description: Learn about advanced hunting queries in Microsoft 365 and how to use them to proactively find threats and weaknesses in your network
+title: Overview - Advanced cyber threat hunting - Microsoft 365 security
```

Comment:
Capability name is "Advanced hunting" so we should not change it to "Advanced cyber threat hunting"